.browserslistrc

```diff
@@ -0,0 +1 @@
+> 0.5%, last 2 versions, Firefox ESR, not dead
```

Comment:
This doesn't quite match our policy: https://github.com/vector-im/element-web#supported-environments is a lot more strict than this

Reply:
Makes sense, I've updated the rules, here's [a list of the matching browsers](https://browsersl.ist/#q=%3E+.5%25%2C+not+ChromeAndroid+%3E+0%2C+not+iOS+%3E+0%2C+not+Samsung+%3E+0%2C+not+OperaMini+%3E+0%2C+not+dead%2C+Firefox+ESR). 
It looks like there's an issue with their tool and OperaMini isn't excluded properly

<img width="681" alt="Screenshot 2022-11-10 at 16 23 05" src="https://user-images.githubusercontent.com/769871/201150893-2850f50f-25d8-4af9-908d-f89e40b163c0.png">